uint16 leds_count
ae_powerboard_control/Color front_left
ae_powerboard_control/Color front_right
ae_powerboard_control/Color rear_left
ae_powerboard_control/Color rear_right

bool enable_add
uint16 leds_add_count
ae_powerboard_control/Color add
---
bool success